src/syntax/abs/tests.rs

```diff
@@ -43,3 +46,28 @@ fn many_decls() {
     };
     assert!(ctx.is_empty());
 }
+
+#[test]
+fn trans_pi_env() {
+    let pi_expr = parse_str_err_printed("val t : ((a : Type) -> (b : Type(a)) -> Type(b));")
+        .unwrap()[0]
+        .clone()
+        .body;
+    let pi_abs = trans_expr(&pi_expr, &[], &BTreeMap::new()).unwrap();
```

Comment:
Using `Default::default()` can eliminate the `use std::collections::btree_map::..` derivative